apps/front/src/components/structure/contractor/mobile-navigation.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <mobile-navigation
+            :items="[
+        {
+            title: 'Mes demandes',
+            value: 'mes-demandes',
```

Comment:
perso ça me va le français mais dans ce cas faut écrire pareil partout. Dans le routeur tu repasse en anglais

Reply:
👌

Reply:
les routes vont rester en anglais, il y a aura la clé to: qui sera le name de la route